### ISSUE TITLE
Specs?

### ISSUE THREAD

**Issue by KCErb** [mixflame/SwissDB#7]
Hi there,

What's your opinion on specs? I could look around and try to see if I can contribute but I know I'd do a better job if someone else setup the context / framework correctly for me.

Do you want to / have the time to get basic specs running that I can build on?

**Comment by mixflame** [comment]
I can try it soon tonight. From the looks of it specs will have to run on
Emulator or Device. Look for a specs/ folder in next push. :)

**Comment by KCErb** [comment]
Nice :+1:

**Comment by mixflame** [comment]
Hey @KCErb 

I just pushed some commits to master which will help set up the test framework. Namely, I created specs/ and specs/helpers/ and a stub spec and a stub helper. Testing looks perfectly possible, but we need to figure out how to set up a dummy RubyMotion Android app inside the main folder, and then use it to run the library, and use (Android)Bacon to test that. But there is a start atleast. If you need I can look into also setting up the app to test against.

Thanks -J

**Comment by KCErb** [comment]
Next chunk of time I get I'll be learning a little more about Travis and Android:

http://docs.travis-ci.com/user/languages/android/

continuous integration would be nice . . .

**Comment by mixflame** [comment]
I agree. CI would be pro. Sweet ! Someone from Bluepotion (Todd) told me
this would be hard. Let's hopefully prove him wrong and pave the way for
Android gem testing!

**Comment by KCErb** [comment]
Proving Todd Werth wrong about something in RubyMotion is a daunting task indeed :fire: 

but I'm game! Let's see what we can do :bowtie:

**Comment by mixflame** [comment]
So. Good news. Our friend @jamonholmgren has fixed the motioneers sign up. So I hope this link will work for you to get to the chat. Thanks @KCErb !!! http://motioneers.herokuapp.com
